frontend/src/App.js

```diff
@@ -70,6 +71,12 @@ class App extends Component {
                   path="/"
                   render={() => <Redirect from="/" to="/inbox" />}
                 />
+                <Route path="/files" component={FileManager} />
+                <Route
+                  exact
+                  path="/"
+                  render={() => <Redirect from="/" to="/files" />}
+                />
```

Comment:
This second `Route` is not needed since there is already a redirect on `/` thats sends the user to `/inbox`